src/frontend/screens/Settings/ProjectSettings/SyncSettings.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import {ScrollView, StyleSheet} from 'react-native';
+import {useIntl, defineMessages} from 'react-intl';
+import {SelectOne} from '../../../sharedComponents/SelectOne';
+import {SYNC_BACKGROUND} from '../../../lib/styles';
+import {
+  usePersistedSettings,
+  usePersistedSettingsAction,
+} from '../../../hooks/persistedState/usePersistedSettings';
+import {SyncActionSheetContent} from './SyncActionSheetContent';
+import {
+  useBottomSheetModal,
+  BottomSheetModal,
+} from '../../../sharedComponents/BottomSheetModal';
+import {SyncSetting} from '../../../sharedTypes';
+
+const m = defineMessages({
+  syncSettingsTitle: {
+    id: 'screens.SyncSettings.title',
+    defaultMessage: 'Sync Settings',
+  },
+  syncPreviews: {
+    id: 'screens.SyncSettings.syncPreviews',
+    defaultMessage: 'Sync Previews (Photos Only)',
+  },
+  syncPreviewsDescription: {
+    id: 'screens.SyncSettings.syncPreviewsDescription',
+    defaultMessage:
+      'Photos will sync at a reduced smaller size. Device will <bold>not</bold> sync audio or video.',
+  },
+  syncEverything: {
+    id: 'screens.SyncSettings.syncEverything',
+    defaultMessage: 'Sync Everything',
+  },
+  syncEverythingDescription: {
+    id: 'screens.SyncSettings.syncEverythingDescription',
+    defaultMessage:
+      'Your device will sync <bold>all</bold> content at full size, including photos, audio, and videos.<newline/><newline/>Note: This will use more storage.',
+  },
+  syncPreviewsBottomSheet: {
+    id: 'screens.SyncSettings.syncPreviewsButtonBottomSheet',
+    defaultMessage: 'Sync Previews?',
+  },
+  syncEverythingBottomSheet: {
+    id: 'screens.SyncSettings.syncEverythingButtonBottomSheet',
+    defaultMessage: 'Sync Everything?',
+  },
+  syncPreviewsDescriptionBottomSheet: {
+    id: 'screens.SyncSettings.syncPreviewsDescriptionBottomSheet',
+    defaultMessage:
+      'Your device will keep all existing data but new observations will sync in a smaller, preview size.<newline/><newline/>You will no longer sync Audio or Video.',
```

Comment:
needs a couple of other changes elsewhere for this to work. would need to add a new custom format e.g.

```ts
newline: () => '\n'
```

to https://github.com/digidem/comapeo-mobile/blob/ee00a488265fefd17f53d6ae59f931467d141332/src/frontend/contexts/IntlContext.tsx#L22-L26



and then every usage of that format has to be in the form of a proper xml-like tag pair e.g. `<newline></newline>` (self-closing tag format does not work, see https://github.com/formatjs/formatjs/issues/1616). 

in order to properly see this working, make sure you do the following:

1. stop the metro server
2. run `npm run extract-messages
3. run `npm start` (which will - importantly - run `npm run build:translations`)

I suspect the disparity we were seeing before in terms of the raw newline characters is related to not having done (2). Extracting the messages via formatjs strips out newline characters (see https://github.com/formatjs/formatjs/issues/3286) which means that the corresponding message in the `translations/` folder doesn't contain the newline characters, hence why i was seeing the issue and you weren't. without the message existing in that directory, the UI will fallback to the message defined in `defaultMessage` in the app code (which contains the newlines).

Reply:
Ok I made them separate text components instead of altering the IntlContext

Reply:
extracted the above into a separate issue: https://github.com/digidem/comapeo-mobile/issues/669